main.cpp

```diff
@@ -180,9 +202,19 @@ struct Restaurant
     void provideEatingUtensils();
 };
 
-void Restaurant::serveFood()
+Restaurant::Restaurant()
 {
+    tblsAvail = 0;
+    workers = 0;
+    platesServed = 0;
+    amFoodPerFamily = 0;
+    hoursWorked = 0;
```

Comment:
This overwrites the default values that your variables have. 
remove these value assignments.
- [ ] revise